.azure/pipelines/ci.yml

```diff
@@ -313,6 +313,22 @@ jobs:
       path: artifacts/logs/
       publishOnError: true
 
+- template: jobs/default-build.yml
+  parameters:
+    condition: ne(variables['SkipTests'], 'true')
+    jobName: Windows_Templates_Test
+    jobDisplayName: "Test: Windows Server Template 2016 x64"
```

Comment:
Is `Test: Windows Server Template 2016 x64` the best name for this Test?  Sounds like you've invented a new edition of Windows Server 2016 called Template. 

Better alternative might be:
`Test: Templates - Windows Server 2016 x64`